### PR TITLE
Improve e2e debuggability

### PR DESCRIPTION
**Pull request by CapacitorSet**
# Description

Adds a couple small features to report more debug information in e2e tests.

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Unit
- [x] E2E

### CODE REVIEW

File: test/e2e/postinstall/basic_test.go [hunk 1]
```diff
@@ -60,6 +64,22 @@ var _ = Describe("Liqo E2E", func() {
...
+						nodeLabel := map[string]string{}
+						nodeLabel[discovery.TenantNamespaceLabel] = "true"
+						var err error
+						tenantNsList, err = cluster.NativeClient.CoreV1().Namespaces().List(ctx, metav1.ListOptions{
+							LabelSelector: labels.SelectorFromSet(nodeLabel).String(),
+						})
+						Expect(err).ToNot(HaveOccurred())
+						return tenantNsList.Items
+					}, timeout, interval).Should(HaveLen(1))
+					Eventually(func() bool {
```
Comment (palexster) [inline test/e2e/postinstall/basic_test.go:78]: This check seems very similar to the one at line 61.

Comment (CapacitorSet) [reply test/e2e/postinstall/basic_test.go:78]: I don't see the similarity, this check tests that the tenant namespace is created while lines 62-66 test that all pods are ready.

Perhaps you mean that lines 62-66 are very similar to lines 78-82? They are, but one checks for pods in the Liqo namespace and the other checks for one pod in the tenant namespace.

**Review by palexster — APPROVED** [review]

**Review by aleoli — APPROVED** [review]

**Pull request merged**